js/components/controlBar.js

```diff
@@ -480,7 +454,7 @@ var ControlBar = createReactClass({
             tooltip={mutedInUi ? CONSTANTS.SKIN_TEXT.UNMUTE : CONSTANTS.SKIN_TEXT.MUTE}
             onClick={this.handleVolumeIconClick}>
           </ControlButton>
-          <VolumeControls {...this.props} />
+          {!this.props.hideVolumeControls ? <VolumeControls {...this.props} /> : null}
```

Comment:
Completely subjective opinion here, but I think the 
`{ condition && <Component /> }`
pattern is more readable. Returning `false` in these statements is valid